packages/cli/lib/services/deploy.service.ts

```diff
@@ -296,8 +290,8 @@ class DeployService {
                     attributes: {},
                     metadata: metadata,
                     input: sync.input || undefined,
-                    // sync_type: sync.sync_type as SyncType,
-                    type: sync.type as any,
+                    sync_type: sync.sync_type,
+                    type: sync.type,
```

Comment:
what's the difference between sync_type and type and why it is uncommented here and not below?

Reply:
- type is action or sync
- sync_type is full or incremental 

it was commented by mistake, but it's not really used in the backend. Those two names are confusing but yeah legacy stuff